app/controllers/shared/api.php

```diff
@@ -74,17 +74,20 @@
         ->from($queueForEvents)
         ->trigger();
 
-    $queueForWebhooks
-        ->from($queueForEvents)
-        ->trigger();
 
-    if ($queueForEvents->getProject()->getId() === 'console') {
-        return;
+    /** Trigger webhooks events only if a project has them enabled */
+    if ($project->getAttribute('webhooks', []) !== []) {
```

Comment:
Do we want to go deeper and check if the specific even is listened by the webhooks? we should have the list of affected events in this context.

Reply:
![Screenshot 2025-01-07 at 10 27 53 AM](https://github.com/user-attachments/assets/216ce5ec-4b4b-4233-8af0-b7325d2e62bd)

I have added a todo here, for additional optimisation. As it needs some benchmarks etc to understand the added latency in a request.